tokens/src/tests_fungibles.rs

```diff
@@ -287,6 +287,62 @@ fn fungibles_unbalanced_trait_should_work() {
 		});
 }
 
+#[test]
+fn fungibles_balanced_done_deposit_emits_event() {
+	ExtBuilder::default()
+		.balances(vec![(ALICE, DOT, 100)])
+		.build()
+		.execute_with(|| {
+			<Tokens as fungibles::Balanced<_>>::done_deposit(DOT, &ALICE, 10);
```

Comment:
this test isn't that useful. should do `deposit` and test the balance and event.

Reply:
https://github.com/open-web3-stack/open-runtime-module-library/pull/931/commits/4c851e507c7e3b2b6527aa30a2e9402ab6f22655